siriuspy/siriuspy/orbintlk/main.py

```diff
@@ -58,6 +58,15 @@ def __init__(self, tests=True):
         self._thread_cb = None
 
         # devices and connections
+        self._fout_devs = {
```

Comment:
I don't know if like this way of defining the FOUTs and their configs. The right way is looking at the timing tree and finding out which FOUTs and which ports are connected to BPMs.

The same comment applies to the EVG RxEnbl config definition.

Reply:
I just pushed a commit with a initial implementation of your suggestion. Please take a look!

Reply:
I just pushed a new commit with your offline suggestions!